app/_layout.tsx

```diff
@@ -127,6 +136,20 @@ function RootLayoutNav() {
           <Stack.Screen name='SellerOptions' options={{ headerShown: false }} />
           <Stack.Screen name='Send' options={{ headerShown: false }} />
           <Stack.Screen name='CompostReport' options={{ headerShown: false }} />
+          <Stack.Screen
+            name='SendAmount'
+            options={{
+              header: () => <StepsComponentCopy steps={steps} />,
+              headerShown: false,
+            }}
+          />
+          <Stack.Screen
+            name='SendReason'
+            options={{
+              header: () => <StepsComponentCopy steps={steps} />,
+              headerShown: false,
```

Comment:
I think you will have to change this so we can see the header
```suggestion
              headerShown: true,
```

Reply:
I leave it  false intentionally cuz you said we will open 2 PR one for separation the amount and reason and the second for the steps so that’s why it’s false